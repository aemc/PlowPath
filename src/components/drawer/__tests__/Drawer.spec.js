import React from "react";
import Component from "../Drawer";
import { shallow } from "enzyme";

const defaultProps = {
  classes: {},
  theme: {}
};

describe("Drawer", () => {
  it("renders", () => {
    const wrapper = shallow(<Component {...defaultProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
