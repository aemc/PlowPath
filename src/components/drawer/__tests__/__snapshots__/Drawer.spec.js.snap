// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(MiniDrawer)
    classes={Object {}}
    theme={Object {}}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "classes": Object {
        "appBar": "MiniDrawer-appBar-1",
        "appBarShift": "MiniDrawer-appBarShift-2",
        "content": "MiniDrawer-content-9",
        "drawer": "MiniDrawer-drawer-5",
        "drawerClose": "MiniDrawer-drawerClose-7",
        "drawerOpen": "MiniDrawer-drawerOpen-6",
        "hide": "MiniDrawer-hide-4",
        "menuButton": "MiniDrawer-menuButton-3",
        "toolbar": "MiniDrawer-toolbar-8",
      },
      "theme": Object {},
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "classes": Object {
          "appBar": "MiniDrawer-appBar-1",
          "appBarShift": "MiniDrawer-appBarShift-2",
          "content": "MiniDrawer-content-9",
          "drawer": "MiniDrawer-drawer-5",
          "drawerClose": "MiniDrawer-drawerClose-7",
          "drawerOpen": "MiniDrawer-drawerOpen-6",
          "hide": "MiniDrawer-hide-4",
          "menuButton": "MiniDrawer-menuButton-3",
          "toolbar": "MiniDrawer-toolbar-8",
        },
        "theme": Object {},
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;

exports[`Drawer renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(MiniDrawer)
    classes={Object {}}
    theme={Object {}}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "classes": Object {
        "appBar": "MiniDrawer-appBar-1",
        "appBarShift": "MiniDrawer-appBarShift-2",
        "content": "MiniDrawer-content-9",
        "drawer": "MiniDrawer-drawer-5",
        "drawerClose": "MiniDrawer-drawerClose-7",
        "drawerOpen": "MiniDrawer-drawerOpen-6",
        "hide": "MiniDrawer-hide-4",
        "menuButton": "MiniDrawer-menuButton-3",
        "toolbar": "MiniDrawer-toolbar-8",
      },
      "theme": Object {},
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "classes": Object {
          "appBar": "MiniDrawer-appBar-1",
          "appBarShift": "MiniDrawer-appBarShift-2",
          "content": "MiniDrawer-content-9",
          "drawer": "MiniDrawer-drawer-5",
          "drawerClose": "MiniDrawer-drawerClose-7",
          "drawerOpen": "MiniDrawer-drawerOpen-6",
          "hide": "MiniDrawer-hide-4",
          "menuButton": "MiniDrawer-menuButton-3",
          "toolbar": "MiniDrawer-toolbar-8",
        },
        "theme": Object {},
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
